styleguide/source/assets/scss/04-templates/_three_column.scss

```diff
@@ -0,0 +1,68 @@
+* {
```

Comment:
Can you explain why this rule is here? Is it resetting something?

Reply:
This was added before we had reset.css incorporated into SG2